frontend/src/documents/DocumentPreview.svelte

```diff
@@ -33,6 +33,8 @@
   {/await}
 {:else if imageExtensions.includes(extension)}
   <img src={url} alt={filename} />
+{:else if ["html", "htm"].includes(extension)}
+  <iframe src={url} title={filename}></iframe>
```

Comment:
You might consider also setting the `iframe` attribute `sandbox=""`. This will prevent the embedded HTML file from doing all sorts of malicious things.

More info here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe

Reply:
done

Reply:
Thanks! 👍